### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/directives/articles/cbd-add-new-article.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<script>
+    export default {
+        name:'cbdAddNewArticle',
+        props: {
+            tags 		: { type: Array  , required: false, default:[]           }, // [] of tag id's
+            customTags 	: { type: Array  , required: false, default:[]           }, // [] of customTag id's
+            adminTags 	: { type: Array  , required: false, default:[]           }, // [] of adminTag text
+            target		: { type: String , required: false, default:'_self'      },
+            buttonText	: { type: String , required: false, default:'Add Article'},
+            cssClass	: { type: String , required: false, default:''           },
```
Comment (sbilodeau) [inline app/directives/articles/cbd-add-new-article.vue:16]: FYI by default attributes are automatically transfered to template root object unless you explicitly block it

File: app/directives/articles/cbd-add-new-article.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<template>
+     <a :href="newArticleUrl" :target="target" :class="cssClass">
+        {{buttonText||'Add article'}}
```
Comment (sbilodeau) [inline app/directives/articles/cbd-add-new-article.vue:3]: can use default slot instead
```suggestion
        <slot>'Add article'</slot>
```

in you other code 
<add-article>My button text</add-article>

File: app/directives/articles/cbd-add-new-article.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+                const domain = window.location.hostname.replace(/[^\.]+\./, '');
+				let baseUrl = 'https://cbd.int/management'
+
+				if(domain=='localhost' || domain == 'cbddev.xyz')
+            		baseUrl = 'https://oasis.cbddev.xyz';
+
+				const queryString = 'tags='		+ (this.tags||[]).join(',') +
+									 '&customTags='	+ (this.customTags||[]).join(',') +
+									 '&adminTags='	+ (this.adminTags||[]).join(',');
```
Comment (sbilodeau) [inline app/directives/articles/cbd-add-new-article.vue:29]: you have to encodeUriComponent values

you can also use string templating `` `tags=${encodeURIComponent(...)}` `` 

```suggestion
				const queryString = 'tags='		+ encodeURIComponent((this.tags||[]).join(',')) +
									 '&customTags='	+ encodeURIComponent((this.customTags||[]).join(',')) +
									 '&adminTags='	+ encodeURIComponent((this.adminTags||[]).join(','));
```

File: app/views/virtual-tables/index.html [hunk 4]
```diff
@@ -292,6 +292,7 @@ <h3>{{virtualArticle.title.en}}</h3>
             </script>
 
             <script type="text/ng-template" id="filterButtons">
+                <cbd-add-new-article ng-vue="vueOptions" v-if="filterAdminTags" :admin-tags="filterAdminTags" button-text="Add Publication/Events" class="pull-right" css-class="btn btn-default"></cbd-add-new-article>
```
Comment (sbilodeau) [inline app/views/virtual-tables/index.html:295]: If you implement above chnage (class + slot) 
```suggestion
                <cbd-add-new-article ng-vue="vueOptions" v-if="filterAdminTags" :admin-tags="filterAdminTags" class="pull-right" class="btn btn-default">Add Publication/Events</cbd-add-new-article> 
```

File: app/directives/articles/cbd-add-new-article.vue (hunk 3 above)
Comment (blaisefonseca) [reply app/directives/articles/cbd-add-new-article.vue]: fixed

File: app/views/virtual-tables/index.html (hunk 4 above)
Comment (blaisefonseca) [reply app/views/virtual-tables/index.html]: fixed

File: app/directives/articles/cbd-add-new-article.vue (hunk 2 above)
Comment (blaisefonseca) [reply app/directives/articles/cbd-add-new-article.vue]: done

File: app/directives/articles/cbd-add-new-article.vue (hunk 1 above)
Comment (blaisefonseca) [reply app/directives/articles/cbd-add-new-article.vue]: done

**Review by randyhoulahan — APPROVED** [review]

File: app/directives/articles/cbd-add-new-article.vue [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<template>
+     <a :href="newArticleUrl" :target="target" :class="cssClass">
```
Comment (sbilodeau) [inline app/directives/articles/cbd-add-new-article.vue:2]:
```suggestion
     <a :href="newArticleUrl" :target="target">
```

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**